charts/ingress-nginx/templates/admission-webhooks/validating-webhook.yaml

```diff
@@ -25,10 +25,9 @@ webhooks:
     sideEffects: None
     admissionReviewVersions:
       - v1
-      - v1beta1
     clientConfig:
       service:
         namespace: {{ .Release.Namespace }}
         name: {{ include "ingress-nginx.controller.fullname" . }}-admission
-        path: /extensions/v1beta1/ingresses
+        path: /networking/v1beta1/ingresses
```

Comment:
is this still v1beta1? My short googling failed me in finding that answer.

Reply:
v1 is available since v1.19.